src/pybind/mgr/dashboard/frontend/src/app/shared/enum/components.enum.ts

```diff
@@ -1,5 +1,5 @@
 export enum Components {
-  auth = 'Login',
+  auth = 'Log in',
```

Comment:
In this case, I think the rationale is just the opposite @nizamial09 : here everything is a noun (component), so Login (Component) makes more sense. Do we need this too?

Reply:
I think no. I made the changes. Thanks @epuertat